src/apps/opening-hours/OpeningHours.entry.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import GuardedApp from "../../components/guarded-app";
+import { GlobalEntryTextProps } from "../../core/storybook/globalTextArgs";
+import { withConfig } from "../../core/utils/config";
+import { withText } from "../../core/utils/text";
+import { withUrls } from "../../core/utils/url";
+import OpeningHours from "./OpeningHours";
+
+export interface OpeningHoursEntryProps {
+  branchId: number;
+  initialDate?: Date;
+}
+
+const OpeningHoursEntry: React.FC<
+  OpeningHoursEntryProps & GlobalEntryTextProps
+> = ({ branchId, initialDate = new Date() }) => {
+  return (
+    <GuardedApp app="opening-hours">
+      <OpeningHours branchId={branchId} initialDate={initialDate} />
```

Comment:
Please do not wrap the OpeningHours component in a Guarded App.

In general guarded apps are meant to ensure a workflow where a end user can execute a request but only if they are authenticated. If they are not they will be redirected to the login system. If they authenticate they are redirected and the original request is executed. Examples of these include adding a material to your favourites or reserving a material.

It is not needed for opening hours.

Reply:
Yeah of course, this obviously shouldn't be guarded.